frontend/src/utils/usePropeller.ts

```diff
@@ -0,0 +1,19 @@
+import create from 'zustand'
+
+interface PropellerState {
+  maxAirSpeed: number
+  cruiseAltitude: number
+  diameterType: string
+  setMaxAirSpeed: (value: number) => void
+  setCruiseAltitude: (value: number) => void
+  setDiameterType: (value: string) => void
+}
+
+export const usePropellerStore = create<PropellerState>()((set) => ({
+  maxAirSpeed: 120,
+  cruiseAltitude: 3,
+  diameterType: 'optimized',
```

Comment:
You can also keep the value of the option to be uppercase, as it'll simplify the logic